eng/doc/DeveloperGuide.md

```diff
@@ -165,19 +167,68 @@ Then, when you run:
 git go-patch extract
 ```
 
-The `go-patch` command will generate a patch file under the `go/patches` directory. The patch file will be prefixed with a serial number (one greater than the number of existing patch files), followed by a dash-separated commit message.
+The `go-patch` command will generate a patch file under the `go/patches` directory.
```

Comment:
For consistency:

```suggestion
The `go-patch extract` command will generate a patch file under the `go/patches` directory.
```

Reply:
Hmm, didn't look at these, IMO they should all also include `git`.